apps/server/src/modules/room/api/mapper/room-invitation-link.mapper.ts

```diff
@@ -0,0 +1,28 @@
+import { RoomInvitationLinkProps } from '../../domain/do/room-invitation-link.do';
+import { RoomInvitationLinkResponse } from '../dto/response/room-invitation-link.response';
+
+export class RoomInvitationLinkMapper {
+	public static mapToRoomInvitationLinkResponse(link: RoomInvitationLinkProps): RoomInvitationLinkResponse {
+		const response = new RoomInvitationLinkResponse({
+			id: link.id,
+			title: link.title,
+			restrictedToCreatorSchool: link.restrictedToCreatorSchool,
+			isOnlyForTeachers: link.isOnlyForTeachers,
+			activeUntil: link.activeUntil,
+			requiresConfirmation: link.requiresConfirmation,
+			roomId: link.roomId,
+			creatorUserId: link.creatorUserId,
+			creatorSchoolId: link.creatorSchoolId,
+		});
+
+		return response;
+	}
+
+	public static mapToRoomInvitionLinksResponse(links: RoomInvitationLinkResponse[]): RoomInvitationLinkResponse[] {
```

Comment:
didn't we fix that yesterday morning?

Reply:
I can't remember and do not understand the question.

Reply:
inviTAtion